src/subgraph/ens_data.py

```diff
@@ -23,12 +23,12 @@
 }
 
 load_dotenv()
-w3 = Web3(Web3.HTTPProvider(f"https://mainnet.infura.io/v3/{os.environ['INFURA_KEY']}"))
+w3 = Web3(Web3.HTTPProvider("https://rpc.ankr.com/eth"))
```

Comment:
I think in practice it's nice to let users of this script chose their Web3 provider endpoint. i agree that it shoudn't be an INFURA_KEY, but can we make it optionally configurable via RPC_URL or the like and only use ankr in case nothing else is defined.

Reply:
Will do.